base/bucket_gocb.go

```diff
@@ -1360,6 +1360,39 @@ func (bucket *CouchbaseBucketGoCB) DeleteWithXattr(k string, xattrKey string) er
 
 }
 
+func (bucket *CouchbaseBucketGoCB) GetXattr(k string, xattrKey string, xv interface{}) (casOut uint64, err error) {
+
+	worker := func() (shouldRetry bool, err error, value interface{}) {
+		res, lookupErr := bucket.Bucket.LookupInEx(k, gocb.SubdocDocFlagAccessDeleted).
+			GetEx(xattrKey, gocb.SubdocFlagXattr).Execute()
+
+		switch lookupErr {
+		case nil:
+			res.Content(xattrKey, xv)
+			cas := uint64(res.Cas())
+			return false, nil, cas
+
+		default:
+			shouldRetry = isRecoverableGoCBError(lookupErr)
+			return shouldRetry, lookupErr, uint64(0)
+		}
+
+	}
+
+	err, result := RetryLoop("GetXattr", worker, bucket.spec.RetrySleeper())
```

Comment:
Can we include key in the worker description like in `GetWithXattr`, but make sure we redact it ahead of time too. The others should be redacting but aren't yet.

```suggestion
	description := fmt.Sprintf("GetXattr %s", UD(k).Redact())
	err, result := RetryLoop(description, worker, bucket.spec.RetrySleeper())
```